src/processor/tree/tree_wrapper.rs

```diff
@@ -69,6 +69,10 @@ impl TreeWrapper<'static> {
 
         root_hash
     }
+
+    pub fn latest_root_hash(&self) -> RootHash {
+        self.tree.latest_root_hash()
+    }
```

Comment:
I don't like the idea of tacking on more methods than needed on the processor and making the internal tree struct public. I'm thinking of making something like a query-only wrapper around the local storage instead, any objections?

Reply:
No, sounds like a good idea :+1: